app/views/collections/edit.html.erb

```diff
@@ -15,9 +15,39 @@
 <% end %>
 
 <div>
-  <%= form_for @collection do |f| %>
-    <%= f.label :name %>
-    <%= f.text_field :name %>
-    <%= f.submit %>
+    <%= form_with model: @collection, local: true, method: :patch, url: collection_path(@collection) do |f| %>
+
+      <%- collections_topics_lookup = @collection
```

Comment:
Would be nice at some point to extract this functionality into a helper method. Perhaps even one that works for topics as well.

Reply:
I've not made my own helper methods before - will have a read and come back to you with any questions